src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationMiddleware.cs

```diff
@@ -40,6 +40,18 @@ public class CookieAuthenticationMiddleware : AuthenticationMiddleware<CookieAut
             {
                 Options.CookieManager = new ChunkingCookieManager(urlEncoder);
             }
+            if (!Options.LoginPath.HasValue)
+            {
+                Options.LoginPath = CookieAuthenticationDefaults.LoginPath;
+            }
+            if (!Options.LogoutPath.HasValue)
+            {
+                Options.LogoutPath = CookieAuthenticationDefaults.LogoutPath;
```

Comment:
Should a logout redirect still be optional? SignOut("Cookies") primary job is to delete the cookie.